### PR TITLE
Disable failing testFixture_TEST for MacOS

### PR DESCRIPTION
**Pull request by Crola1702**
# 🦟 Bug fix

Disables failing test from #1023 

## Checklist
- [ ] Signed all commits for DCO
- [ ] Added tests
- [ ] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [ ] Consider updating Python bindings (if the library has them)
- [ ] `codecheck` passed (See [contributing](https://gazebosim.org/docs/all/contributing#contributing-code))
- [ ] All tests passed (See [test coverage](https://gazebosim.org/docs/all/contributing#test-coverage))
- [ ] While waiting for a review on your PR, please help review [another open pull request](https://github.com/pulls?q=is%3Aopen+is%3Apr+user%3Agazebosim+archived%3Afalse+) to support the maintainers

**Note to maintainers**: Remember to use **Squash-Merge** and edit the commit message to match the pull request summary while retaining `Signed-off-by` messages.

### CODE REVIEW

**Review by Blast545 — APPROVED** [review]
Can you add a link to the original issue as a comment in the code? 

LGTM, I think it's reasonable to wait for someone in the dev team to review before merging.

**Comment by Blast545** [comment]
Also cc: @scpeters for awareness. We are disabling it for now, but we'll start tracking the tests disabled by the buildfarm team and ping the team/community for help on those every now and then.

File: python/CMakeLists.txt
Comment (iche033) [inline python/CMakeLists.txt:89]: is it #1887?

Comment (Crola1702) [reply python/CMakeLists.txt:89]: You are right. My bad

**Review by iche033 — APPROVED** [review]

**Pull request merged**
